common/secrets.js

```diff
@@ -102,10 +102,14 @@ const POSTCODES_API_KEY =
 
 const S3_KMS_KEY_ID = process.env.S3_KMS_KEY_ID || getSecret('s3.kms.key.id');
 
+const DATA_STUDIO_AFA_URL =
+    process.env.DATA_STUDIO_AFA_URL || getSecret('dataStudio.url.awardsForAll');
```

Comment:
We probably don't need the `process.env.DATA_STUDIO_AFA_URL` part as it's unlikely we'd need to override this in the environment or CI.